packages/commonwealth/webpack/webpack.base.config.js

```diff
@@ -60,6 +60,9 @@ module.exports = {
         process.env.FLAG_PROPOSAL_TEMPLATES
       ),
     }),
+    new webpack.DefinePlugin({
```

Comment:
Can we use the FLAG_ prefix for the environment var, to maintain consistency?

Reply:
nice catch, thanks!